crates/transaction-emitter-lib/src/emitter/submission_worker.rs

```diff
@@ -189,7 +238,7 @@ impl SubmissionWorker {
                 }
             }
             Err(uncommitted) => {
-                let num_uncommitted = uncommitted.len() as u64;
+                let num_uncommitted = uncommitted.values().sum::<usize>() as u64;
```

Comment:
this was a bug - we were understating uncommitted transactions.
If a full batch for account fails, we report 1 failed, and rest succeeded. this means if all fail, failure rate reported would've been only 1/batch_size